packages/ytube/src/lib/youtubeClient.ts

```diff
@@ -1,202 +1,296 @@
-import { youtube_v3 } from '@googleapis/youtube';
-import { OAuth2Client, TokenInfo } from 'google-auth-library';
-import ytdl from 'ytdl-core';
-import Schema$PlaylistItem = youtube_v3.Schema$PlaylistItem;
-import Schema$Channel = youtube_v3.Schema$Channel;
-import { Channel, DomainError, Result, User, Video, YoutubeAuthorizationFailed } from '@youtube-sync/domain';
-import { Readable } from 'stream';
-import { statsRepository } from '..';
-import R from 'ramda'
-import { GetTokenResponse } from 'google-auth-library/build/src/auth/oauth2client';
+import { youtube_v3 } from '@googleapis/youtube'
+import { OAuth2Client } from 'google-auth-library'
+import ytdl from 'ytdl-core'
+import Schema$PlaylistItem = youtube_v3.Schema$PlaylistItem
+import Schema$Channel = youtube_v3.Schema$Channel
+import { Channel, ChannelVerificationFailed, ExitCodes, User, Video } from '@youtube-sync/domain'
+import { Readable } from 'stream'
+import { statsRepository } from '..'
+
+// YPP induction criteria, each channel should meet following criteria
+const MINIMUM_SUBSCRIBERS_COUNT = 50
+const MINiMUM_VIDEO_COUNT = 10
```

Comment:
Typo, should be `MINIMUM`, all caps

Reply:
addressed in [5c475a3](https://github.com/Joystream/youtube-synch/pull/40/commits/5c475a30e429375f666b84d4b368e5387e68ee42)